src/components/layouts/Layout.ts

```diff
@@ -0,0 +1,25 @@
+import './common.scss';
+
+import '../Head';
+import {Block} from '../../modules/Block';
+
+export class Layout extends Block {
+  constructor(props: Record<string, unknown>) {
+    super(props);
+    this.on('preparing', () => {
+      const title = document.querySelector('title') as HTMLElement;
```

Comment:
Лучше избегать явного приведения типов
```suggestion
      const title = document.querySelector<HTMLElement>('title');
```